pyfr/integrators/base.py

```diff
@@ -156,22 +156,6 @@ def _init_reg_banks(self):
                            for em in self.system.ele_banks])
             )
 
-    def _get_kernels(self, name, nargs, **kwargs):
-        # Transpose from [nregs][neletypes] to [neletypes][nregs]
-        transregs = zip(*self._regs)
-
-        # Generate an kernel for each element type
-        kerns = proxylist([])
```

Comment:
See if this lets us eliminate some other proxylist uses.

Reply:
pintgkernels is the last use of proxylist in integrators (except multip)

Reply:
I think it can just be a regular list as I do not think we rely on any specific proxylist functionality.